packages/flutter/lib/src/widgets/table.dart

```diff
@@ -397,6 +397,9 @@ class _TableElement extends RenderObjectElement {
 /// the [TableCell] widget to its enclosing [Table] must contain only
 /// [TableRow]s, [StatelessWidget]s, or [StatefulWidget]s (not
 /// other kinds of widgets, like [RenderObjectWidget]s).
+///
+/// To create an empty [TableCell], provide a [Container] or [SizedBox.shrink]
```

Comment:
While both work, SizedBox.shrink will be every so slightly more efficient. I would only mention that one and remove "Container".